test/service/test_request_builder.py

```diff
@@ -301,24 +292,16 @@ def test_create_request_terms_and_missing_values(self):
                             "constant_score": {
                                 "filter": {
                                     "terms": {
-                                        "term2.keyword": [
-                                            "test"
-                                        ]
+                                        "contents.projects.institutions.keyword": ["test"]
                                     }
                                 }
                             }
                         },
                         {
                             "constant_score": {
                                 "filter": {
-                                    "bool": {
-                                        "must_not": [
-                                            {
-                                                "exists": {
-                                                    "field": "term3.keyword"
-                                                }
-                                            }
-                                        ]
+                                    "terms": {
+                                        "contents.specimens.disease.keyword": ["__null__"]
```

Comment:
This is what I'd expect.

Reply:
Fixed the `entity_id` test. `TestRequestBuilder` uses its own test `request_config` which has `entity_id` as a valid field, however the filter values won't be `None` translated unless the field has a `field_types` entry. Fixed by replacing `entity_id` with a more real example `libraryConstructionApproach`